x-pack/plugins/lens/public/xy_visualization/xy_config_panel/threshold_panel.tsx

```diff
@@ -270,6 +351,34 @@ export const ThresholdPanel = (
           }}
         />
       </EuiFormRow>
+      {currentYConfig?.icon && currentYConfig.icon !== 'none' && (
+        <EuiFormRow
+          display="columnCompressed"
+          fullWidth
+          label={i18n.translate('xpack.lens.xyChart.thresholdMarker.position', {
+            defaultMessage: 'Icon position',
+          })}
+        >
+          <EuiButtonGroup
+            isFullWidth
+            legend={i18n.translate('xpack.lens.xyChart.thresholdMarker.position', {
+              defaultMessage: 'Icon position',
+            })}
+            data-test-subj="lnsXY_markerPosition_threshold"
+            name="markerPosition"
+            buttonSize="compressed"
+            options={getIconPositionOptions({
+              isHorizontal,
+              axisMode: currentYConfig!.axisMode,
+            })}
+            idSelected={`${idPrefix}${currentYConfig?.iconPosition || 'auto'}`}
+            onChange={(id) => {
+              const newMode = id.replace(idPrefix, '') as IconPosition;
+              setYConfig({ forAccessor: accessor, iconPosition: newMode });
+            }}
+          />
+        </EuiFormRow>
+      )}
```

Comment:
Rather than hiding this `EuiFormRow` until an icon is selected, let's instead show the `EuiFormRow`, but disable the `EuiButtonGroup` when no icon is selected. While disabled, hovering the `EuiButtonGroup` should show a tooltip containing a message explaining why it is disabled, like:

```
You must select an icon in order to alter its position.
```

If the user selects an icon, the `EuiButtonGroup` should become enabled to allow the user to change positioning. This suggested behavior is more consistent with how we've been treating conditional form elements recently.